### PR TITLE
Network fees

### LINKED ISSUE #584: network fee (vortex) - indicate the fee distribution breakdown to users
As a user, I would like to better understand the fees I'm earning as part of my staking in Bancor's pools
## High level ##
Following the DAO vote to increase vortex fees, we want to reflect the impact to our LPs on the frontend and better explain what portion of the fees is used for the vortex vs LP value appreciation.

### calculation ###
/stats api: totalLpFees24h = totalFees24h - totalNetworkFees24h 
/pools api: lpFees24h = fees24h - networkFees24h
/bnt api: same calculation as /pools api

### Places  ###
1. Earn page **V3 table**
- APR (7d) -> update the tooltip text.
-- current: Estimated APR based on the last 7d trading fees
-- updated: Estimated APR based on the last 7d LP fees
- APR (7d) -> updated calculation
-- current: we are using "fees7d" to calculate the APR
-- updated: replace "fees7d" with "lpFees7h"
```lpFees7h = fees7d - networkFees7d```
- token icon tooltip (mouse over the token icon)
-- Add one more row at the bottom of the tooltip for lpFees24h 
-- Add (I) icon next to the "lpFees24h" that leads to - https://vote.bancor.network/#/proposal/0x61cd3ec4295bf0819fb7fd80a9dcc0922036510f40b02c76213b29c48a1a5137
-- Mockup - https://www.figma.com/file/fwADI9wqDrRAdlMX8EddCw/Bancor-v3?node-id=12678%3A385521
Screen Shot 2022-07-27 at 10 59 40

2. Earn page **V2 table**
- APR -> update tooltip text
-- current: Estimated APR based on the last 24h trading fees
-- updated: Estimated APR based on the last 24h LP fees
- APR -> updated calculation
-- replace "fees7d" with "lpFees7d"

3. Earn page **total stats**
- Fees 24h -> add tooltip with fee breakdown
-- https://www.figma.com/file/fwADI9wqDrRAdlMX8EddCw/Bancor-v3?node-id=12678%3A383753

4. Earn page **top performing** ticker
- confirm the APR we calculate and display is using "lpFees"

5. Portfolio page **deposit and earn** section
- confirm the APR we calculate and display is using "lpFees"

6. deposit flow
- confirm the APR we calculate and display is using "lpFees"

### PR DESCRIPTION
**Pull request by zavelevsky**
fixes #584

### CODE REVIEW

**Comment by Yanivshechter1** [comment]
Please add the (i) icon with the link to "https://vote.bancor.network/#/proposal/0x61cd3ec4295bf0819fb7fd80a9dcc0922036510f40b02c76213b29c48a1a5137"
for the following tooltips:

- [ ] V3 APR tooltip 

- [ ] V2 APR tooltip 

- [ ] Fees breakdown tooltip in total stats

File: src/services/observables/pools.ts [hunk 1]
```diff
@@ -187,8 +188,15 @@ const buildPoolV3Object = async (
   const standardRewardsApr24H = standardsRewardsAPR(apiPool, programs);
   const standardRewardsApr7d = standardsRewardsAPR(apiPool, programs);
 
-  const tradingFeesApr24h = calcApr(apiPool.fees24h.usd, stakedBalance.usd);
-  const tradingFeesApr7d = calcApr(apiPool.fees7d.usd, stakedBalance.usd, true);
+  const tradingFeesApr24h = calcApr(
+    new BigNumber(apiPool.fees24h.usd).minus(apiPool.networkFees24h.usd),
```
Comment (pingustar) [inline src/services/observables/pools.ts:192]: might be worth using the toBigNumber helper

**Review by pingustar — APPROVED** [review]
LGTM

**Comment by ashachaf** [comment]
- [ ] earn table v3
flow
- mouse over APR
- click the (i) in the tool tip
- return to the app tab
actual: all pools are sorted by APR
expected: my click on the (i) should not cause the sorting to change

**Review by Yanivshechter1 — APPROVED** [review]

**Pull request merged**